_posts/news/2025-01-23-announcing-scalajs-1.18.2.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: Announcing Scala.js 1.18.2
+category: news
+tags: [releases]
+permalink: /news/2025/01/23/announcing-scalajs-1.18.2/
+---
+
+
+We are pleased to announce the release of Scala.js 1.18.2!
+
+This is mostly a hotfix release for a binary incompatibility present in 1.18.0 and 1.18.1 and affecting some libraries built with Scala.js < 1.11.
```

Comment:
Maybe add the same note here that 1.18.0 / 1.18.1 did *not* produce bad artifacts?